dotcom-rendering/src/client/main.editionsCrossword.tsx

```diff
@@ -1,14 +1,7 @@
 /* eslint-disable ssr-friendly/no-dom-globals-in-module-scope */
-// @ts-expect-error: Cannot find module
-import css from '@guardian/react-crossword/lib/index.css';
+import type { CAPICrossword } from '@guardian/react-crossword/dist/@types/CAPI';
```

Comment:
Non-blocking: we should probably avoid importing from this directory. In the article crossword player we have something like this to get access to this type:

```tsx
import type { CrosswordProps } from '@guardian/react-crossword';

type CAPICrossword = CrosswordProps['data'];
```